sig-network/annual-report-2021.md

```diff
@@ -0,0 +1,36 @@
+# 2021 SIG network annual report
+
+This report reflects back on CY 2020 and was written in April 2021.
+
+> Note: Each response should have supporting documentation, linked KEPs, graphs, data
+> Note: Err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference. 
+
+# Operational
+
+- How are you doing with operational tasks in [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)?
+  - Is your [README](./README.md) accurate? 
+  - Have a CONTRIBUTING.md file?
+  - All subprojects correctly mapped and listed in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sig-list.md)?
+  - What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+
+- How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+- Same question as above but for working groups.
+- When was your last monthly community-wide update? (provide link to deck and/or recording)
+
+# Membership
+
+- Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+- How do you measure membership? By mailing list members, OWNERs, or something else?
+- How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+- Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
```

Comment:
Regular triage is probably the most we do at the moment - basically new members can volunteer to help walk a user through gathering information about a potential bug / enhancement and thus classify the issue. 

So, not a ton.

Reply:
Over the summer we encouraged new members by helping beginners run triage in the meetings.